src/app/components/elements/ChangePassword.jsx

```diff
@@ -262,11 +264,11 @@ export default reduxForm(
             }))
         },
         notify: (message) => {
-            dispatch({type: 'ADD_NOTIFICATION', payload: {
+            dispatch(appActions.addNotification({
                 key: "chpwd_" + Date.now(),
                 message,
-                dismissAfter: 5000}
-            });
+                dismissAfter: 5000,
```

Comment:
gonna take a while to get used to dangling commas... are those allowed by our style-guide?

Reply:
they're not disallowed: https://github.com/steemit/condenser/blob/master/.eslintrc#L17

Reply:
and airbnb/javascript wants them https://github.com/airbnb/javascript#commas--dangling

we should enforce dangling commas; they make diffs way easier to read.